ski_conditions/apps/app_scraping/tests/test_commands.py

```diff
@@ -0,0 +1,125 @@
+import os
+
+import pytest
+import responses
+from django.core import management
+
+from ..management.commands.do_scraping import (
+    HeavenlyScraper, HeavenlySnowReport, KirkwoodScraper, KirkwoodSnowReport, NorthstarScraper, NorthstarSnowReport
+)
+from ..models import SkiResort
+
+
+# pylint: disable=no-member
+@responses.activate
+@pytest.mark.django_db()
+def test_scraping():
+    '''
+    test lift and trail number scraping
+    '''
+
+    assert SkiResort.objects.count() == 0
+
+    # use saved html instead of accessing live site
+    body_Kirkwood = open(os.path.join(os.path.dirname(__file__), 'data', 'kirkwood.html'))
+    responses.add(responses.GET, KirkwoodScraper.url, status=200, body=body_Kirkwood.read())
+
+    body_Heavenly = open(os.path.join(os.path.dirname(__file__), 'data', 'heavenly.html'))
+    responses.add(responses.GET, HeavenlyScraper.url, status=200, body=body_Heavenly.read())
+
+    body_Northstar = open(os.path.join(os.path.dirname(__file__), 'data', 'northstar.html'))
+    responses.add(responses.GET, NorthstarScraper.url, status=200, body=body_Northstar.read())
+
+    body_Kirkwood_snow = open(os.path.join(os.path.dirname(__file__), 'data', 'kirkwood_snow.html'))
+    responses.add(responses.GET, KirkwoodSnowReport.url, status=200, body=body_Kirkwood_snow.read())
+
+    body_Heavenly_snow = open(os.path.join(os.path.dirname(__file__), 'data', 'heavenly_snow.html'))
+    responses.add(responses.GET, HeavenlySnowReport.url, status=200, body=body_Heavenly_snow.read())
+
+    body_Northstar_snow = open(os.path.join(os.path.dirname(__file__), 'data', 'northstar_snow.html'))
+    responses.add(responses.GET, NorthstarSnowReport.url, status=200, body=body_Northstar_snow.read())
+
+    # call do_scraping command
+    management.call_command('do_scraping')
+
+    assert SkiResort.objects.count() == 3
+
+    # assert trail and lift values for Kirkwood, Heavenly, Northstar
+    kirkwood = SkiResort.objects.get(resort_name='Kirkwood')
+    assert kirkwood.acres_open == 1031
+    assert kirkwood.terrain_percent == 45
+    assert kirkwood.trails_open == 52
+    assert kirkwood.total_trails == 88
+    assert kirkwood.lifts_open == 7
+    assert kirkwood.total_lifts == 14
+
+    heavenly = SkiResort.objects.get(resort_name='Heavenly')
+    assert heavenly.acres_open == 22
+    assert heavenly.terrain_percent == 0
+    assert heavenly.trails_open == 4
+    assert heavenly.total_trails == 97
+    assert heavenly.lifts_open == 4
+    assert heavenly.total_lifts == 28
+
+    northstar = SkiResort.objects.get(resort_name='Northstar')
+    assert northstar.acres_open == 1182
+    assert northstar.terrain_percent == 37
+    assert northstar.trails_open == 57
+    assert northstar.total_trails == 100
+    assert northstar.lifts_open == 10
+    assert northstar.total_lifts == 20
+
+
+# pylint: disable=no-member
+@responses.activate
+@pytest.mark.django_db()
+def test_snow_report():
+    '''
+    test snow report scraping
```

Comment:
This comment doesn’t tell us anything we don’t already know from the method name. It can be removed.

Reply:
Removed comment